src/components/Post/GovernanceSideBar/Referenda/Modal/VoteSuccessModal.tsx

```diff
@@ -44,7 +51,7 @@ const VoteInitiatedModal = ({
 	multisig,
 	balance,
 	conviction,
-	title,
+	// title,
```

Comment:
Please update that title, instead of commenting that, also update that for multisig